custom/icds_reports/tests/outputs/ccs_record_monthly_2017-05-01_sorted.csv

```diff
@@ -81,22 +81,22 @@ a12,dd3ff191-1acd-4c78-a96f-a026c870d6bc,2017-05-01,545,other,1,1,,0,0,0,0,0,0,0
 a12,e413c896-7595-4a11-8d16-79c87cad3423,2017-05-01,305,lactating,1,1,,16,1,0,0,0,0,0,0,,0,0,0,0,0,1,0,0,0,,0,0,0,0,0,0,0,0,0,0,0,0,0,0,,0,0,1,1,0,2017-03-01,0,obc,no,no,yes,,,,,,,,,,,,,,,,2017-08-08,,,,,,,,,,1,,,,,,,2017-04-12,,1992-01-10,0,f76921f9-ea4f-4c4c-b6f6-de5752102e4e
 a12,eccae16c-866c-43d8-8021-c5d9238794c7,2017-05-01,257,lactating,1,1,,16,1,0,0,0,0,0,0,,0,0,0,0,0,1,0,0,0,,0,0,0,0,0,0,0,0,0,0,1,1,0,1,,0,0,1,1,0,2017-03-01,0,obc,no,no,yes,,,,,,,,,,,,1,1,,,2017-08-08,,,,,,,,,,1,,,,,,,2017-04-12,,1996-01-08,0,ee80d394-995e-447c-912b-9e681e53f2e6
 a12,ed7e32da-5a16-4606-812a-ba30d208ba66,2017-05-01,281,other,1,1,,0,0,0,0,0,0,0,0,,0,0,0,0,0,1,0,0,0,,0,0,0,0,0,0,0,0,0,0,0,0,0,0,,0,0,0,0,0,2016-05-01,0,obc,no,no,yes,,,,,,,,,,,,,,,,2017-08-08,,,,,,,,,,0,,,,,,,2017-04-12,,1994-01-08,0,ce57fb59-aa4c-4665-8d7b-8ff89e1ce8ad
-a12,f2600795-0236-45a7-ac6c-baff437681f4,2017-05-01,810,other,1,1,,0,0,0,0,0,0,0,0,,0,0,0,0,0,1,0,0,0,,0,0,0,0,0,0,0,0,0,0,0,0,0,0,,0,0,0,0,0,2016-05-01,0,obc,no,no,yes,,,,,,,,,,,,,,,,2017-08-08,,,,,,,,,,0,,,,,,,2017-04-12,,1950-01-28,0,d65b5b3b-266c-4d5d-b571-494b6023a2e3
+a12,f2600795-0236-45a7-ac6c-baff437681f4,2017-05-01,809,other,1,1,,0,0,0,0,0,0,0,0,,0,0,0,0,0,1,0,0,0,,0,0,0,0,0,0,0,0,0,0,0,0,0,0,,0,0,0,0,0,2016-05-01,0,obc,no,no,yes,,,,,,,,,,,,,,,,2017-08-08,,,,,,,,,,0,,,,,,,2017-04-12,,1950-01-28,0,d65b5b3b-266c-4d5d-b571-494b6023a2e3
```

Comment:
There were some inconsistency in the age_in_month.
is age_in_month in '2017-04-01' is x then in ''2017-05-01' has to be x+1.
fixed in this commit